TESTING.md

```diff
@@ -0,0 +1,61 @@
+# Testing on testnet
+
+## Creating a gift wallet
+
+[Download node-funder](https://github.com/ethersphere/node-funder) which will be used to create gift wallets.
+
+Make sure your Go version is 1.18 or above. Run `go version` to verify.
+
+Set some environment variables:
+
+```bash
+export TESTNET_PRIVATE_KEY="..."
+export TESTNET_RPC_ENDPOINT="https://goerli.infura.io/v3/..."
+export NODE_FUNDER_PATH="<path to node-funder>"
+```
+
+Make sure you have the latest utility CLI version: `npm install --global cafe-tui`
```

Comment:
You don't have to `npm install` when you use `npx` as it will prompt the user to install the package if it won't find it ;-) 
```suggestion
```